assets/scripts/cat-api.js

```diff
@@ -0,0 +1,13 @@
+this.baseUrl = 'https://api.thecatapi.com/v1/';
+
+async function getAllCatBreeds(){
+    let response = null;
+    try {
+        response = await axios.get(`${this.baseUrl}breeds`);
```

Comment:
Same here, this `this` is not correct.

`await axios.get(`${baseUrl}breeds`);`